sources/packages/web/src/views/aest/AssessmentsSummary.vue

```diff
@@ -11,7 +11,11 @@
     :applicationId="applicationId"
     @viewStudentAppeal="goToStudentAppeal"
   />
-  <HistoryAssessment :applicationId="applicationId" />
+  <HistoryAssessment
+    :applicationId="applicationId"
+    @viewStudentAppeal="goToStudentAppeal"
+    @viewScholasticStandingChange="goToScholasticStanding"
```

Comment:
Where is goToScholasticStanding defined?

Reply:
Okay. Just got confused when I saw that.

Reply:
I added the implementation and then removed it but forgot to remove that line. It is better without it that's why I removed 😉